config/spring19_seminar/cola_var/sample.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+#SBATCH --time=12:00:00
+#SBATCH --mem=20000
+#SBATCH --gres=gpu:1
+#SBATCH --job-name=myrun
+#SBATCH --output=slurm_%j.out
+
+python main.py --config_file cola_confs/cola_bert.conf \
```

Comment:
cola_confs or cola_var?

Reply:
cola_confs is the folder I store those config files on my laptop